src/main/java/com/flightstats/hub/dao/aws/S3LargeContentDao.java

```diff
@@ -270,15 +270,15 @@ public void delete(String channel) {
     }
 
     public static class S3LargeContentDaoBuilder {
```

Comment:
Aw, this kind of makes me sad. We usually reserve builders for plain old domain objects, not stuff with logic in it. On top of that, I think this could be lombok-ified, but that would just make the outer class even weirder.

Reply:
wait til you see ChannelConfig :(:(:(

Reply:
agreed :(